docs/workflows.rst

```diff
@@ -592,3 +561,22 @@ A visualization of the AROMA component classification is also included in the HT
     Right hand side of each map: time series (top in seconds),
     frequency spectrum (bottom in Hertz).
     Components classified as signal in green; noise in red.
+
+
+.. _bold_t2s:
+
+T2* Driven Coregistration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+:mod:`fmriprep.workflows.bold.t2s.init_bold_t2s_wf`
+
+If multi-echo :abbr:`BOLD (blood-oxygen level-dependent)` data is supplied,
+this workflow uses the `tedana`_ `T2* workflow`_ to generate an adaptive T2* map
+and optimally weighted combination of all supplied single echo time series.
+This optimaly combined time series is then carried forward for all subsequent
+preprocessing steps.
+Optionally, if the ``--t2s_coreg`` flag is supplied, the T2* map is then used
```

Comment:
`t2s-coreg` (hyphen, not underscore)

https://github.com/poldracklab/fmriprep/blob/8a4de9fccd28d06ce765df1a5aea4e4abb9ed0f2/fmriprep/cli/run.py#L117